app/src/main/java/org/oppia/app/testing/BindableAdapterTestViewModel.kt

```diff
@@ -1,15 +1,15 @@
 package org.oppia.app.testing
 
 import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.ViewModel
 import org.oppia.app.fragment.FragmentScope
 import org.oppia.app.model.TestModel
+import org.oppia.app.viewmodel.ObservableViewModel
 import javax.inject.Inject
 
 // TODO(#59): Make this view model only included in relevant tests instead of all prod builds.
 /** A [ViewModel] for testing the bindable RecyclerView adapter. */
```

Comment:
Change this comment too.

Reply:
done

Reply:
Do I need to also update the comment in the files that I didn't change?

Reply:
No that should be fine. Just fix it in all the files that you have changed.